pkg/core/chain/chain.go

```diff
@@ -113,6 +113,19 @@ func (c *Chain) Listen() {
 	}
 }
 
+// LaunchConsensus will listen for an init message, and send a round update
+// once it is received.
+func (c *Chain) LaunchConsensus() {
+	initChan := make(chan *bytes.Buffer, 1)
+	id := c.eventBus.Subscribe(msg.InitializationTopic, initChan)
+	roundBuf := <-initChan
+	round := binary.LittleEndian.Uint64(roundBuf.Bytes())
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	c.sendRoundUpdate(round, c.prevBlock.Header.Seed, c.prevBlock.Header.Hash)
```

Comment:
Instead of getting round from `msg.InitializationTopic`,  would it make sense to use `c.prevBlock.Header.Height + 1` ?

Reply:
Ah, good point. Would save us the need to actually read the buffer. Will adjust it